bots/pr/src/main/java/org/openjdk/skara/bots/pr/JEPCommand.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package org.openjdk.skara.bots.pr;
+
+import org.openjdk.skara.forge.PullRequest;
+import org.openjdk.skara.issuetracker.Comment;
+import org.openjdk.skara.issuetracker.Issue;
+import org.openjdk.skara.issuetracker.jira.JiraProject;
+
+import java.io.PrintWriter;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+public class JEPCommand implements CommandHandler {
+    static final String JEP_LABEL = "jep";
+    private static final String jepMarker = "<!-- jep: '%s' '%s' '%s' -->"; // <!-- jep: 'JEP-ID' 'ISSUE-ID' 'ISSUE-TITLE' -->
+    static final Pattern jepMarkerPattern = Pattern.compile("<!-- jep: '(.*?)' '(.*?)' '(.*?)' -->");
+    private static final String unneededMarker = "<!-- jep: 'unneeded' 'unneeded' 'unneeded' -->";
+
+    private void showHelp(PrintWriter reply) {
+        reply.println("""
+                Command syntax:
+                 * `/jep <issue-id>`
+                 * `/jep JEP-<jep-id>`
+                 * `/jep jep-<jep-id>`
+                 * `/jep unneeded`
+
+                Some examples:
+
+                 * `/jep JDK-1234567`
+                 * `/jep 1234567`
+                 * `/jep jep-123`
+                 * `/jep JEP-123`
+                 * `/jep unneeded`
+
+                Note:
+                The project prefix of the issue ID (`JDK-` in the above examples) is optional.
+                But the `JEP-` or `jep-` prefix is required when you provide a JEP ID.
+                And the issue type must be `JEP`.
+                """);
+    }
+
+    private Optional<Issue> getJepIssue(String args, PullRequestBot bot) {
+        Optional<Issue> jbsIssue;
+        if (args.startsWith("jep-") || args.startsWith("JEP-")) {
```

Comment:
```suggestion
        if (args.toUpperCase().startsWith("JEP-")) {
```

Then variations like "Jep-123" is also accepted.

Reply:
Fixed and added a test case.